custom_components/sat/config_flow.py

```
@@ -457,6 +470,35 @@ async def async_step_pid_controller(self, _user_input: dict[str, Any] | None = N
+            await coordinator.async_will_remove_from_hass()
+
+        options = []
+        for name, _info in MANUFACTURERS.items():
+            manufacturer = ManufacturerFactory.resolve_by_name(name)
+            options.append({"value": name, "label": manufacturer.name})
+
+        return self.async_show_form(
+            last_step=True,
+            step_id="manufacturer",
+            data_schema=vol.Schema({
+                vol.Required(CONF_MANUFACTURER, default=default_manufacturer): selector.SelectSelector(
```

Ruff (0.8.2)

`CONF_MANUFACTURER` may be undefined, or defined from star imports

(F405)